praw/__init__.py

```diff
@@ -1610,9 +1610,8 @@ def set_settings(self, subreddit, title, public_description='',
                      submit_link_label='', submit_text_label='',
                      exclude_banned_modqueue=False, comment_score_hide_mins=0,
                      public_traffic=False, collapse_deleted_comments=False,
-                     spam_comments='low', spam_links='high',
-                     spam_selfposts='high', submit_text='',
-                     hide_ads=False, **kwargs):
+                     spam_comments='low', spam_links='high', spam_selfposts='high',
+                     submit_text='', hide_ads=False, suggested_comment_sort, **kwargs):
```

Comment:
@13steinj  I'll fix it - That reply of his came in when I was in Star Wars and I didn't see his edit. Thanks for catching it.